supporter-product-data/cloudformation/cfn.yaml

```
@@ -345,3 +345,55 @@ Resources:
       Statistic: Sum
       TreatMissingData: notBreaching
     DependsOn: SupporterProductData
+
+  CsvReadAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: CreateProdResources
+    Properties:
+      AlarmActions:
+        - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:reader-revenue-dev
+      AlarmName: !Join
+        - ' '
+        - - 'URGENT 9-5 -'
+          - !Ref 'Stage'
+          - 'There was a csv read failure when loading supporter product data into DynamoDB'
+      AlarmDescription: !Join
+        - ' '
+        - - 'Impact - On or more subscriber will not get their digital benefits'
+          - 'Follow the process in https://docs.google.com/document/d/1_3El3cly9d7u_jPgTcRjLxmdG2e919zCLvmcFCLOYAk/edit'
+      MetricName: CsvReadFailure
+      Namespace: supporter-product-data
+      Dimensions:
+        - Name: Environment
+          Value: PROD
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      Threshold: 1
+      Period: 60
+      EvaluationPeriods: 1
+      Statistic: Average
```

minor as the threshold is 1 anyway but
```suggestion
      Statistic: Sum
```